morphocut/str.py

```diff
@@ -6,7 +6,24 @@
 
 @Output("string")
 class Format(Node):
-    """Format a string just like :py:meth:`str.format`."""
+    """
+    Format a string just like :py:meth:`str.format`.
+    
+    Args:
+        fmt (str): A format in which we want our string to be.
+        *args: Arguments to be replaced with placeholders in fmt
+        _args: Arguments to be appended after *args
+        _kwargs: Key value paired arguments
+        **kwargs: Key value paired arguments to be appended after _kwargs
+
+    if we supply the same key twice, once in _kwargs and once in kwargs, then the values in those keys will be overwritten.
+
+    Example:
+        .. code-block:: python
+
+            result = Format(fmt, *args, _args=_args, _kwargs=_kwargs, **kwargs)()
```

Comment:
This example is now correctly formatted but shows nothing more than the function signature. Please write a complete example with Pipeline and so on that a user could run and get meaningful output.